web/thesauruses/c/17/control_structures.json

```diff
@@ -0,0 +1,79 @@
+{
+  "meta": {
+    "language": "c",
+    "language_version": "17",
+    "language_name": "C",
+    "structure": "control_structures"
+  },
+  "concepts": {
+    "if_conditional": {
+      "code": "if (condition) {\n    statements;\n}",
+      "name": "If conditional"
+    },
+    "if_else_conditional": {
+      "code": "if (condition) {\n    statements;\n} else {\n    statements;\n}",
+      "name": "If/Else conditional"
+    },
+    "if_elseif_conditional": {
+      "code": "if (condition) {\n    statements;\n} else if (condition) {\n    statements;\n}",
+      "name": "If/ElseIf conditional"
+    },
+    "if_elseif_else_conditional": {
+      "code": "if (condition) {\n    statements;\n} else if (condition) {\n    statements;\n} else {\n    statements;\n}",
+      "name": "If/ElseIf/Else conditional"
+    },
+    "switch_statement": {
+      "code": "switch (expression) {\n  case x:\n    statements;\n    break;\n  case y:\n    statements;\n    break;\n  default:\n    statements;\n}",
+      "comment": "Expression is compared with cases using the strict === comparator",
```

Comment:
C doesn't have a `===` operator. Are you thinking of JavaScript? You could just say the types have to match and it's a strict comparison.